rollup.config.js

```diff
@@ -24,9 +26,18 @@ export default {
       ]
     }
   ],
-  external: [],
+  // eslint-disable-next-line
+  // https://github.com/formium/tsdx/blob/462af2d002987f985695b98400e0344b8f2754b7/src/createRollupConfig.ts#L63
+  external: (id) => {
+    // bundle in polyfills as TSDX can't (yet) ensure they're installed as deps
+    if (id.startsWith('regenerator-runtime')) {
```

Comment:
Peaks.js has no dependency on regenerator-runtime, so this isn't needed. The external list can be more simply created using [rollup-plugin-peer-deps-external](https://github.com/pmowrer/rollup-plugin-peer-deps-external).

Reply:
There's a few plugins that do this, but TSDX's simple function does the same job. We can use either.